### ISSUE TITLE
remove dictFromVector3 from code, replace with .dictionary()

### ISSUE THREAD

**Issue by blairmacintyre** [mozilla-mobile/webxr-ios#125]
There are a few places where old code (dictFromVector3) is used to convert some to an object with (x, y, z) values, appears similar to the .dictionary() overrides on the object.

I think these can be replaced, but don't want to fiddle with that right now.

**Comment by blairmacintyre** [comment]
(I suppose this may actually be the objC version of the same thing?)

**Comment by robomex** [comment]
You're correct there are a couple lingering usages of dictFromVector3 due to a bit of ObjC being left.  Converting the remaining ObjC has proven tricky and taken a back seat recently as my focus has been on other items.
